src/pages/ResetPassword/index.tsx

```
@@ -2,101 +2,160 @@ import type React from "react"
-      setSuccess("Password changed successfully. Redirecting...")
-      setTimeout(() => navigate("/login"), 2500)
-    } catch (err: any) {
-      setError(err?.response?.data?.message || "An error occurred.")
+        try {
+            const response = await axios.post("http://localhost:3000/auth/reset-password", {
+                token,
+                password: newPassword,
+            })
+            setSuccess("Password changed successfully. Redirecting...")
+            setTimeout(() => navigate("/login"), 2500)
+        } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)